client/src/components/Status.js

```diff
@@ -0,0 +1,58 @@
+import { HeartIcon, ChatIcon, ShareIcon } from '@heroicons/react/outline'
+import React, { useState } from 'react'
+import { Link } from 'react-router-dom'
+import Comments from './Comments'
+import MovieShow from './MovieShow'
+
+
+const Status = ({ post }) => {
+	const [visibleComment, setVisibleComment] = useState(false)
+	
+	const makeCommentsVisible = () => {
```

Comment:
No matter what type of status it is, the first 2 comments will be supplied along the API response and it should be visible. When user clicks, 'Show more comments' type button, all comments will show.
Anyway, you don't have to do this now. We can do that in a separate PR in future